### PR TITLE
Fixes #474 - Allow disabling HTTP altogether

### LINKED ISSUE #474: SSL is not enforced if SSL settings are provided
If SSL is enabled via the `ssl_keystore_path` flag, SSL works via https://localhost:8443 but it is not enforced; it's still possible to visit the UI via the unencrypted http://localhost:8080. If gone unnoticed, you might submit your username/password via plain text over HTTP.

* Is it possible to enable HTTP -> HTTPS redirection if SSL is enabled?

### PR DESCRIPTION
**Pull request by kolloch**
by bumping up chaos version and documenting the --http_disable flag.

(Will not build until we publish new Chaos release)

### CODE REVIEW

File: docs/docs/command-line-flags.md [hunk 1]
```diff
@@ -106,13 +106,16 @@ The Web Site flags control the behavior of Marathon's web site, including the us
...
     packaged JAR.
-* `--http_address` (Optional. Default: None): The address on which to listen
+* `--http_address` (Optional. Default: all interfaces): The address on which to listen
     for HTTP requests.
 * `--http_credentials` (Optional. Default: None): Credentials for accessing the
     HTTP service in the format of `username:password`. The username may not
     contain a colon (:). May also be specified with the `MESOSPHERE_HTTP_CREDENTIALS` environment variable. 
 * `--http_port` (Optional. Default: 8080): The port on which to listen for HTTP
     requests.
+* `--http_disable` (Optional.): Disable HTTP completely. This is only allowed if you configure HTTPS.
```
Comment (guenter) [inline docs/docs/command-line-flags.md]: `--disable-http`

**Comment by guenter** [comment]
:+1:

**Comment by ConnorDoyle** [comment]
Thanks @kolloch, published the new Chaos version yesterday and re-started the build.  LGTM!

**Comment by ConnorDoyle** [comment]
D'oh!  The requisite patches hadn't yet landed in Chaos...

**Comment by ConnorDoyle** [comment]
Now they have been, and released `chaos-0.6.5`.

**Pull request merged**